hazelcast/src/test/java/com/hazelcast/sql/SqlWithoutSqlModuleTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.core.JetTestSupport;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.annotation.ParallelJVMTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class SqlWithoutSqlModuleTest extends JetTestSupport {
+
+    @Test
+    public void test() {
```

Comment:
Don't think that would be possible to verify precisely. The problem is that if we add it in hazelcast-sql where its possible to spin up such instances (that is with actual SQL engine), the test itself will already have SQL in the classpath. If we add in core module - it will be exactly this test essentially.

Reply:
This is important change for an existing user and will break his/her application until he/she make the classpath changes for the new dependency, right?

Reply:
It was like this before. You need to add hazelcast-sql to the member's classpath to execute queries, even in 5.2